### PR TITLE
Game handler and activity

### PR DESCRIPTION
**Pull request by VictorSchneuwly**
Create a game from the lobby and players join it. The phone of the host is the one monitoring the game.

### CODE REVIEW

**Comment by VictorSchneuwly** [comment]
For now it works online on firebase but the phone app is not updating for some reason.

**Review by nathanduchesne — APPROVED** [review]

File: app/src/main/java/com/github/fribourgsdp/radio/GameHandler.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    protected fun executeOnUpdate(): EventListener<DocumentSnapshot> {
+        return EventListener<DocumentSnapshot> { snapshot, e ->
+            if (e != null) {
+                view.displayError("An error occurred")
+            }
+
+            this.handleSnapshot(snapshot)
+
+        }
+    }
```
Comment (shanzhaiii) [inline app/src/main/java/com/github/fribourgsdp/radio/GameHandler.kt:21]: documentation would be nice

**Review by shanzhaiii — APPROVED** [review]
looks good, just lacking a bit of documentation as its a bit complex

File: app/src/main/java/com/github/fribourgsdp/radio/GameHandler.kt (hunk 1 above)
Comment (VictorSchneuwly) [reply app/src/main/java/com/github/fribourgsdp/radio/GameHandler.kt:21]: It'll be done next week !

**Pull request merged**